static/js/narrow.js

```diff
@@ -124,42 +124,47 @@ export function set_narrow_title(title) {
     notifications.redraw_title();
 }
```

Comment:
I think it makes sense to inline `set_narrow_title` now, like this; I'll just apply that change before merging:

``` diff
diff --git a/static/js/narrow.js b/static/js/narrow.js
index f653bed325..1a5455bbc0 100644
--- a/static/js/narrow.js
+++ b/static/js/narrow.js
@@ -116,13 +116,8 @@ export function save_pre_narrow_offset_for_reload() {
     }
 }
 
-export let narrow_title = "home";
 export let has_shown_message_list_view = false;
 
-function set_narrow_title(title) {
-    narrow_title = title;
-}
-
 export function compute_narrow_title(filter) {
     if (filter === undefined) {
         // "All messages" and "Recent conversations" views have
@@ -170,9 +165,9 @@ export function compute_narrow_title(filter) {
     return filter_title;
 }
 
+export let narrow_title = "home";
 export function update_narrow_title(filter) {
-    const narrow_title_string = compute_narrow_title(filter);
-    set_narrow_title(narrow_title_string);
+    narrow_title = compute_narrow_title(filter);
     notifications.redraw_title();
 }
 
```